azure/placeholder-function-app-template.json

```diff
@@ -179,7 +183,7 @@
                         "value": "[variables('resourceGroupName')]"
                     },
                     "subnetResourceId": {
-                        "value": "[reference('function-app-subnet').outputs.SubnetResourceId.value]"
+                        "value": "[reference(concat(variables('functionAppName'), '-subnet-', parameters('utcValue'))).outputs.SubnetResourceId.value]"

```

Comment:
this might need to be changed to what we agree on the name of the subnet deployment to be